client/layout/style.scss

```diff
@@ -72,135 +115,140 @@
 	}
 }
 
-.layout__primary {
-	transition: all 0.15s ease-in-out;
+// Setup the secondary element, which contains the sidebar and
+// the site-selector elements.
+.layout__secondary {
+	position: fixed;
+		top: 47px;
+		left: 0;
+		bottom: 0;
+	color: var( --sidebar-color );
+	background: var( --sidebar-background );
+	border-right: 1px solid darken( $sidebar-bg-color, 5% );
+	width: $sidebar-width-max;
+	overflow: hidden;
 
-	.is-section-preview & {
-		height: 100%;
+	@include breakpoint( "<960px" ) {
+		width: $sidebar-width-min;
 	}
-}
 
-.layout__content a {
-	text-decoration: none;
-}
+	@include breakpoint( "<660px" ) {
+		width: 100%;
+	}
 
-.layout {
-	.sidebar,
-	.layout__secondary .site-selector,
-	.current-site,
-	.sidebar__menu {
-		transform: translateX( 0 );
-		transition: all 0.15s cubic-bezier(0.075, 0.820, 0.165, 1.000);
+	// The editor has its own sidebar which doesnt
+	// use the secondary element.
+	.is-section-post-editor & {
+		display: none;
 	}
 }
 
-.layout {
-	.sidebar,
-	.masterbar {
-		@media print {
-			display: none;
+// Setup the site-selector element. Its default
+// position is off screen.
```

Comment:
Why offscreen? Is there some JS that needs node dimensions? If not can we `display: none`?

Reply:
If we `display: none;` then we can't do the transitions.

Reply:
Feels like we should still be able to accomplish this, and not need to render misc elements. If it's too tricky with CSS, we can always do add a JS transition. CSS animation appears to set @nylen's comp on 🔥 anyway.

Reply:
For what it's worth, it's the continuous CSS animations that really slow things down for me.  Quick transitions are fine.  Though I agree we shouldn't render elements offscreen for transitions, instead we can use a bit of JS that renders the element when the transition should start.

Reply:
I'll go ahead and defer on this for now. I suppose we might run into the trap of needing to add animation classes and then may get into buggy states if we fail to remove them.